committing.rst

```diff
@@ -428,10 +428,9 @@ Prefix the backport pull request with the branch, for example::
    
 Note that cherry_picker.py_ adds the branch prefix automatically.
    
-Apply the label ``cherry-pick or X.Y`` to the backport pull request.  Once the backport
-pull request has been created, remove the ``needs backport to X.Y`` label from the
-original pull request on ``master``.  Only Core Developers can apply labels to GitHub
-pull requests.  
+Once the backport pull request has been created, remove the
+``needs backport to X.Y`` label from the original pull request on ``master``.
+Only Core Developers can apply labels to GitHub pull requests.
```

Comment:
Might as well put this in parentheses to de-emphasize it as people familiar with GH will know this.